content/en/docs/reference/command-line-tools-reference/feature-gates/local-storage-capacity-isolation-fs-quota-monitoring.md

```diff
@@ -9,11 +9,14 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.15"
+    toVersion: "1.30"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.31"
 ---
-When `LocalStorageCapacityIsolation`
-is enabled for
-[local ephemeral storage](/docs/concepts/configuration/manage-resources-containers/)
-and the backing filesystem for [emptyDir volumes](/docs/concepts/storage/volumes/#emptydir)
-supports project quotas and they are enabled, use project quotas to monitor
-[emptyDir volume](/docs/concepts/storage/volumes/#emptydir) storage consumption rather than
-filesystem walk for better performance and accuracy.
+When `LocalStorageCapacityIsolation` 
+is enabled for 
+[local ephemeral storage](/docs/concepts/configuration/manage-resources-containers/), 
+the backing filesystem for [emptyDir volumes](/docs/concepts/storage/volumes/#emptydir) supports project quotas,
+and `UserNamespacesSupport` is enabled, 
+project quotas are used to monitor [emptyDir volume](/docs/concepts/storage/volumes/#emptydir) storage consumption rather than using filesystem walk, ensuring better performance and accuracy.
```

Comment:
nit: wrap the long line, and remove the extra hyperlink.

```suggestion
project quotas are used to monitor emptyDir volume storage consumption rather than
using filesystem walk, ensuring better performance and accuracy.
```